src/lemlib/logger/logger.cpp

```diff
@@ -0,0 +1,14 @@
+
+#include "logger.hpp"
```

Comment:
Include full path in file include
```suggestion
#include "lemlib/logger/logger.hpp"
```